docs/octopus-rest-api/octopus.server.exe-command-line/path.md

```diff
@@ -21,7 +21,7 @@ Where [<options>] is any of:
                                with caution
       --cacheDirectory=VALUE Directory to use for temporary files and cachin-
                                g, e.g. downloaded packages. The data in this
-                               directory can be removed when the Octopus Server
+                               directory can be removed when the Octopus server
```

Comment:
🤔 This should be capitalised... This commit https://github.com/OctopusDeploy/OctopusDeploy/commit/0cf3fb7f71c02d273be5c20aa0922c6d1961edf0 should've shipped in 2020.1.1?